benchmarks/similarity_scores_time_benchmark.py

```diff
@@ -62,19 +68,19 @@ def benchmark(strat='k-means', limit=50000, n_proto=100, hash_dim=None, ngram_ra
                                     hashing_dim=hash_dim, n_prototypes=n_proto, random_state=3498)
 
     column_trans = ColumnTransformer(
-        transformers=transformers + [('sim_enc', sim_enc, ['Description'])],
+        transformers=transformers + [('sim_enc', sim_enc, ['description'])],
         remainder='drop'
     )
 
     t0 = time()
-    X = column_trans.fit_transform(df)
+    X_enc = column_trans.fit_transform(X_b)
     t1 = time()
     t_score_1 = t1 - t0
 
     model = pipeline.Pipeline([('logistic', linear_model.LogisticRegression())])
```

Comment:
I realize that this is not in the code that you changed, but I don't understand why we need a pipeline with only a single model in it. We could just use this model, couldn't we?

Reply:
Yes, works fine without the pipeline. I updated it.

Reply:
Thanks